Realm/RLMRealm.mm

```diff
@@ -276,7 +276,12 @@ + (void)setDefaultRealmPath:(NSString *)defaultRealmPath {
     s_defaultRealmPath = defaultRealmPath;
 }
 
-+ (NSString *)writeablePathForFile:(NSString*)fileName
++ (NSString *)writeableTemporaryPathForFile:(NSString *)fileName
+{
+    return [NSTemporaryDirectory() stringByAppendingPathComponent:fileName];
```

Comment:
Apple's docs don't say why `URLForDirectory` is preferable, but I suspect it's because it exposes more functionality, which we don't need in this case, so I'd say it's safe to use `NSTemporaryDirectory()` here.